src/coldfront_plugin_api/urls.py

```diff
@@ -31,7 +33,12 @@ def get_queryset(self):
 urlpatterns += [
     path('scim/v2/Groups', groups.ListGroups.as_view()),
     path('scim/v2/Groups/<int:pk>', groups.GroupDetail.as_view()),
-    path('scim/v2/Users', users.ListUsers.as_view()),
-    path('scim/v2/Users/<str:username>', users.UserDetail.as_view())
+    re_path(r'^scim/v2$',
```

Comment:
I am unclear why we have replaced the `path(...)` expressions here with `re_path(...)` expressions, which are generally harder to read. Couldn't this be expressed something like:

```
path('scim/v2', scim_views.SCIMView.as_view(implemented=False), name='root'),
path('scvim/v2/Users', scim_views.UsersView.as_view(), name='users'),
path('scvim/v2/Users/<str:username>', scim_views.UsersView.as_view(), name='users'),
```

Reply:
The view `django_scim.views.UsersView` is[ implemented in a fancy way ](https://github.com/15five/django-scim2/blob/33e7caa0c996d1977cb70a0f7463ba31d5ee24b8/src/django_scim/views.py#L288)where its behavior changes on whether `/<uuid>` was added to the URL or not. it also [hardcodes](https://github.com/15five/django-scim2/blob/33e7caa0c996d1977cb70a0f7463ba31d5ee24b8/src/django_scim/views.py#L37) the url arguement to be called `uuid`.

Reply:
Would it work using `<str:uuid>`, then, or does it somehow require `re_path`?

Reply:
Turns out you're right. The URLs still works fine if I split them and use `path`